GDJS/Runtime/CustomRuntimeObject2D.ts

```diff
@@ -0,0 +1,42 @@
+namespace gdjs {
+  /**
+   * Base class for 2D custom objects.
+   */
+  export class CustomRuntimeObject2D extends gdjs.CustomRuntimeObject {
+    constructor(
+      parent: gdjs.RuntimeInstanceContainer,
+      objectData: ObjectData & CustomObjectConfiguration
+    ) {
+      super(parent, objectData);
+      this.getRenderer().reinitialize(this, parent);
+    }
+
+    protected _createRender(): gdjs.CustomObjectRenderer {
+      const parent = this._runtimeScene;
+      return new gdjs.CustomObjectRenderer(
+        this,
+        this._instanceContainer,
+        parent
+      );
+    }
+
+    protected _reinitializeRender(): void {
+      this.getRenderer().reinitialize(this, this.getParent());
+    }
+
+    getRenderer(): gdjs.CustomObjectRenderer {
+      return super.getRenderer() as gdjs.CustomObjectRenderer;
+    }
+
+    getRendererObject() {
+      return this.getRenderer().getRendererObject();
+    }
+
+    updateFromObjectData(
```

Comment:
Do we really need to redefine this? Can't gdjs.CustomRuntimeObject do this?

I see the parent is doing `return this._instanceContainer.updateFrom(oldObjectData, newObjectData);` - is this normal?